Visualizing.ipynb

```diff
@@ -0,0 +1,334 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import seaborn as sns\n",
+    "import pandas as pd\n",
+    "from wordcloud import WordCloud\n",
+    "from wordcloud import ImageColorGenerator\n",
+    "from PIL import Image\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import requests\n",
+    "from datetime import datetime, timedelta"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Define a function to get data for a specific date\n",
+    "def get_data_for_date(date_str):\n",
+    "    alerts_url = f'https://www.oref.org.il//Shared/Ajax/GetAlarmsHistory.aspx?lang=he&fromDate={date_str}&toDate={date_str}&mode=0'\n",
```

Comment:
"fromDate={date_str}&toDate={date_str}"

Are you sure you wanna lookup alerts for a certain date only? why not accept a range like in my original notebook?

Reply:
Since the "pikud haoref" website provides JSON of up to 2,000 values,
It was done in this way because on the first day there were over 2,000 alerts and this caused information to be lost, so I used this way

Reply:
Oh, it's probably a new thing as when I wrote my original code there was no such limit.
Merged